core/src/Streamly/Internal/Data/Stream/Zip.hs

```diff
@@ -89,3 +90,6 @@ instance Monad m => Applicative (ZipStream m) where
 
     {-# INLINE (<*>) #-}
     ZipStream m1 <*> ZipStream m2 = ZipStream $ Stream.zipWith id m1 m2
+
+getZipStream :: ZipStream m a -> Stream m a
+getZipStream = unZipStream
```

Comment:
It was `getZipSerialM` - https://streamly.composewell.com/haddocks/streamly-0.8.2/Streamly-Internal-Data-Stream-Zip.html#t:ZipSerialM . But looks like that constructor was not exported because the internal stream was StreamK, so there is not point adding getZipStream. Let's remove it. You can revert this commit.